### PR TITLE
fix signature of R.propOr

### LINKED ISSUE #847: propOr documentation re. parameter order vs reality
The documentation states that the parameter order for R.propOr is property name, then defaultValue, then object.

But [in fact](https://github.com/ramda/ramda/blob/master/dist/ramda.js#L4103) it's defaultValue, then propertyName, then object.

### PR DESCRIPTION
**Pull request by davidchambers**
Closes #847

### CODE REVIEW

**Comment by davidchambers** [comment]
Could one of you review this change? :)

File: src/propOr.js [hunk 1]
```diff
@@ -10,9 +10,9 @@ var _has = require('./internal/_has');
  * @func
  * @memberOf R
  * @category Object
- * @sig s -> v -> {s: x} -> x | v
- * @param {String} p The name of the property to return.
+ * @sig a -> String -> Object -> a
  * @param {*} val The default value.
+ * @param {String} p The name of the property to return.
  * @param {Object} obj The object to query.
  * @return {*} The value of given property or default value.
```
Comment (buzzdecafe) [inline src/propOr.js]: should it be:

>  The value of given property *of the supplied object* or *the* default value.

?

Comment (CrossEye) [inline src/propOr.js]: I like the formulation from @buzzdecafe

Comment (davidchambers) [inline src/propOr.js]: Nice tweak. Updated.

**Pull request merged**